articles/install-guide/standalone.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Execute Q# programs without a driver in a host language 
+author: KittyYeungQ
+ms.author: kitty
+ms.date: 4/24/2020
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.update
```

Comment:
This uid should be updated since microsoft.quantum.update is already the uid for the update page. Could be microsoft.quantum.install.standalone?